db/schema.rb

```diff
@@ -305,6 +305,7 @@
     t.integer  "dod_bah"
     t.boolean  "approved",                                            default: false
     t.boolean  "vet_tec_provider",                                    default: false
+    t.boolean  "closure109"
```

Comment:
Where did this come from? It's not reflected in the migration

Reply:
branch not updated with master